source/runner/DangerUtils.ts

```diff
@@ -0,0 +1,13 @@
+export function sentence(array: Array<string>): string {
```

Comment:
I guess my concern here was that these functions don't have an JSDoc, but they correspond directly to the `DangerUtilsDSL` interface definition. I don't want to write the JSDoc twice, so is it cool to leave these functions without JSDoc? If the signature of these functions ever changed and the `DangerUtilsDSL` did not, TypeScript would not compile, so I'm thinking we are 👍  here.